components/TextField/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Platform, StyleSheet, Text, View, TextInput } from 'react-native';
+
+export default class TextField extends React.Component {
+  render() {
+    return (
+    <View style={styles.textField}>
+      {this.props.title && <Text style={styles.titleText}>{this.props.title}</Text>}
```

Comment:
Don't need a title since we are going to have a CritiqueSection that will take a title and an input (such as your TextField)